src/EditorFeatures/Core/StackTraceExplorer/StackTraceExplorerUtilities.cs

```diff
@@ -118,90 +104,25 @@ Task<DefinitionItem> GetDefinitionAsync(IMethodSymbol method)
             }
         }
 
-        private static bool MatchParameters(ImmutableArray<IParameterSymbol> parameters, StackFrameParameterList stackFrameParameters)
+        private static async Task<IMethodSymbol?> TryGetBestMatchAsync(Project project, string fullyQualifiedTypeName, StackFrameSimpleNameNode methodNode, StackFrameParameterList methodArguments, StackFrameTypeArgumentList? methodTypeArguments, CancellationToken cancellationToken)
         {
-            if (parameters.Length != stackFrameParameters.Parameters.Length)
+            var compilation = await project.GetRequiredCompilationAsync(cancellationToken).ConfigureAwait(false);
+            var type = compilation.GetTypeByMetadataName(fullyQualifiedTypeName);
```

Comment:
```suggestion
            var type = compilation.GetBestTypeByMetadataName(fullyQualifiedTypeName);
```

Reply:
it would be great if we could avoid getting the compilation if the final part of the type-name is not found in the index for this project.  getting a compilation is expensive.